### DISCUSSION TITLE
Setup IPv6 properly on docker (Fix Source IPv6 propagation)

### DISCUSSION THREAD

**Discussion by giscus** [FarisZR/FarisZR.com#9] [Comments]
# Setup IPv6 properly on docker (Fix Source IPv6 propagation)

Docker IPv6 isn’t as easy as it seems, exposing docker containers to IPv6 connections without any prior setup will endup with a lot of connections seemingly coming from the Gateway, in this post i explain how you fix IPv6 propagation by Enabling proper IPv6 support in docker.

https://fariszr.com/docker-ipv6-setup-with-propagation/

**Comment by muxhax** [comment]
thank you very much! Enabled ipv6 by your steps (docker compose down, edit yaml with network-default-enable_ipv6) and voila! )
